app/models/service_retire_task.rb

```diff
@@ -68,6 +70,13 @@ def create_task(svc_rsc, parent_service, nh, parent_task)
 
   private
 
+  def retire_subtasks?(parent_svc)
+    if parent_svc.kind_of?(ServiceAnsiblePlaybook) && !parent_svc.retireable?
+      return false
```

Comment:
Does Rubocop complain if this is one line?  
`return false if parent_svc.kind_of?(ServiceAnsiblePlaybook) && !parent_svc.retireable?`